### PR TITLE
db.diagram syntax

### PR DESCRIPTION
**Pull request by wagnerlmichael**
This program takes all txts from a directory (works with those that were created with the script that produces sql code) and edits them so that they are the syntax used by db.diagram. 

It works perfectly for each table **unless** there are multiple primary keys, in which case a slight manual change is needed.

### CODE REVIEW

File: dbdiagram_syntax/dbdiagram_syntax.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    """
+    This is a complementary program to the psql_ops/create_diagram/create_diagram.py file. That file creates
+    table syntax for sql in the form of txt files. This script will edit those txt files
+    so that they are more akin to the syntax of dbdiagram.io.
+
+    This script will collect all txt files in the directory input, and
+    edit them to approximate the syntax for tables in db diagram.
+
+    """
```
Comment (mgutierrezc) [inline dbdiagram_syntax/dbdiagram_syntax.py]: Good docstring! Don't forget to add an Input and output line

File: dbdiagram_syntax/dbdiagram_syntax.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    txts_location = input("Input the path where txt files are stored: ")
+
+    db_diagram_location = input("Input the path where the new dbdiagram txt files will be stored: ")
```
Comment (mgutierrezc) [inline dbdiagram_syntax/dbdiagram_syntax.py]: This could be both arguments from a function

**Review by mgutierrezc — CHANGES_REQUESTED** [review]
The code is good! However, it should be packaged in a function defined before `if __name__ == '__main__':` and called after that line

File: dbdiagram_syntax/dbdiagram_syntax.py (hunk 2 above)
Comment (wagnerlmichael) [reply dbdiagram_syntax/dbdiagram_syntax.py]: I edited the script based on your comments! Let me know what you think

**Review by mgutierrezc — APPROVED** [review]

**Pull request merged**